launch_cameras.sh

```diff
@@ -33,8 +33,16 @@ echo "Reading camera configurations from $json_file..."
 # Define the list of allowed camera names
 # allowed_cameras=("beam_camera_left" "beam_camera_right" "shoulder_camera_left" "shoulder_camera_right" "wrist_camera_left" "wrist_camera_right")
 # allowed_cameras=("shoulder_camera_left" "wrist_camera_left")
+allowed_cameras_file="$DATA/allowed_cameras.json"
+if [ ! -f "$allowed_cameras_file" ]; then
+    echo "Error: File $allowed_cameras_file does not exist."
```

Comment:
Would appreciate an instructive error message like this:
```suggestion
    echo "Error: File $allowed_cameras_file does not exist.
    Please create file: $allowed_cameras_file in this format based on BaseRepo4 camera_msgs.py CameraEnum\'s
        {
   \"allowed_cameras\": [
    \"wrist_camera_left\",
    \"wrist_camera_right\",
    \"handset_wrist_camera\"
  ]
}
"
```